src/main/java/uk/gov/hmcts/darts/casedocument/service/impl/GenerateCaseDocumentForRetentionDateBatchProcessorImpl.java

```diff
@@ -33,11 +32,11 @@ public void processGenerateCaseDocumentForRetentionDate(int batchSize) {
         OffsetDateTime currentTimestamp = currentTimeHelper.currentOffsetDateTime();
         OffsetDateTime caseRetailUntilTimestamp = currentTimestamp.plusDays(caseDocumentExpiryDays);
         OffsetDateTime caseDocumentCreatedAfterTimestamp = currentTimestamp.minusDays(caseDocumentExpiryDays);
-        List<Integer> casesIds = caseRepository.findCasesNeedingCaseDocumentForRetentionDateGeneration(caseRetailUntilTimestamp,
-                                                                                                       caseDocumentCreatedAfterTimestamp,
-                                                                                                       Pageable.ofSize(batchSize));
+        var casesIds = caseRepository.findCasesNeedingCaseDocumentForRetentionDateGeneration(caseRetailUntilTimestamp,
```

Comment:
Why was this changed to var? Not sure it meets the criteria for using var outlined in best practices here: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1694706043#BestPractices(Backend)-Usevarthoughtfully